docs/documentation/zh/declaration-files/By Example.md

```diff
@@ -1,24 +1,21 @@
 ---
-title: Declaration Reference
+title: 声明文件参考
 layout: docs
 permalink: /zh/docs/handbook/declaration-files/by-example.html
-oneline: "How to create a d.ts file for a module"
+oneline: "如何为一个模块创建一个 d.ts 文件"
 ---
 
-The purpose of this guide is to teach you how to write a high-quality definition file.
-This guide is structured by showing documentation for some API, along with sample usage of that API,
-and explaining how to write the corresponding declaration.
+本指南的目的是教你如何编写高质量的声明文件。本指南通过展示某些 API 的文档、API 的示例用法，并解释如何编写相应的声明。
 
-These examples are ordered in approximately increasing order of complexity.
+这些示例大致按照复杂性递增的顺序排列。
 
-## Objects with Properties
+## 带有属性的对象
 
-_Documentation_
+_文档_
```

Comment:
```suggestion
*文档*
```

Reply:
Please use `*` rather than `_` for italic styling

Reply:
I don't see this one and all the similar ones convention in https://github.com/ts-zh-docs/TypeScript-Website/blob/v2/docs/translation-guide.md, could you point the related rule?

Reply:
https://github.com/mdn/translated-content/pull/17745#discussion_r1443566737

The link referenced by @yin1999 is a dead link now. The main point of that is that `_` cannot make Chinese words italic, it can only make English words italic

Reply:
The original changes are correct for the  italics

Reply:
https://stackoverflow.com/a/73475445/15647007